<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
<title>org.junit.rules.ErrorCollector.checkSucceeds(org.junit.rules.ErrorCollectorTest$CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<style type="text/css">
body { background-color:#fff; color:#000; font-family:Helvetica, sans-serif; font-size:10pt }
h1 { font-size:14pt }
.code { border-collapse:collapse; width:100%; font-family: "Andale Mono", monospace; font-size:10pt; line-height: 1.2em }
.color_black { color: black }
.color_blue { color: blue }
.color_green { color: green }
.color_orange { color: orange }
.color_red { color: red }
.line { border-left: 3px solid #ccc; padding-left: 1ex; white-space: pre; }
.msg { background-color:#fff8b4; color:#000; float:left; font-family:Helvetica, sans-serif; font-size: smaller; font-weight: bold; margin-bottom:10px; margin-top:10px; max-width:60em; padding:0.5em 1ex 0.5em 1ex; -webkit-border-radius:5px; -webkit-box-shadow:1px 1px 7px #000; white-space: pre-wrap; word-wrap: break-word; }
.num { background-color:#eee; color:#444; font-size: smaller; padding-right:2ex; text-align:right; width:2.5em; }
.dangling { background-color:white; color: gray; }
.visited { background-color:LemonChiffon; color: darkmagenta; }
.tooltip { display: none; background-color:#FFF0F0; border: 2px solid #F00; font-weight: normal; left:10em; padding: 2px; position: absolute; top: -1em; -webkit-border-radius:5px; -webkit-box-shadow:1px 1px 7px #000; z-index: 1}
.with_tooltip { position: relative; }
.with_tooltip:hover .tooltip, .visited:hover .tooltip { display: block; }
</style>
</head>
<body>
<center><h1>Cfg Node <a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">9</a></h1></center>PROC: <a href="./../org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e.html">Object ErrorCollector.checkSucceeds(ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1)</a> LINE: <a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>
<br>PREDS:
<a href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node8.html"><span class='visited with_tooltip'>N_8<span class='tooltip'>node8 preds:7 succs:9 exn:5 </span></span></a><br>SUCCS:
<a href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node10.html"><span class='visited with_tooltip'>N_10<span class='tooltip'>node10 preds:9 succs:11 exn:5 </span></span></a><br>EXN:
<a href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node5.html"><span class='visited with_tooltip'>N_5<span class='tooltip'>node5 preds:17 succs:2 exn:2 </span></span></a><br>
<LISTING class='color_green'>n$7=*&amp;error:java.lang.AssertionError* [line 93];
_=*n$7:java.lang.AssertionError [line 93];
n$9=*&amp;e:java.lang.Object* [line 93];
n$10=_fun_Throwable AssertionError.initCause(Throwable)(n$7:java.lang.AssertionError*,n$9:java.lang.Object*) virtual [line 93];
*&amp;$irvar1:java.lang.Throwable*=n$10 [line 93];
</LISTING>
<hr width="100%">
<a name="session8" href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9.html#session8">node9#session8</a>(<a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>) [compute pre] liveness 9
<LISTING class='color_black'>JOIN

LEFT:   normal:{ error, this }, exn:{ }
RIGHT:  normal:{ }, exn:{ }
RESULT: normal:{ error, this }, exn:{ }

</LISTING>
<hr width="100%">
<a name="session9" href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9.html#session9">node9#session9</a>(<a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>) [exec] liveness 9
<LISTING class='color_black'>PRE STATE:
normal:{ error, this }, exn:{ }

INSTR=  *&amp;$irvar1:java.lang.Throwable*=n$10 [line 93]; 

STATE:
normal:{ n$10, error, this }, exn:{ }

INSTR=  n$10=_fun_Throwable AssertionError.initCause(Throwable)(n$7:java.lang.AssertionError*,n$9:java.lang.Object*) virtual [line 93]; 

STATE:
normal:{ n$7, n$9, error, this }, exn:{ }

INSTR=  n$9=*&amp;e:java.lang.Object* [line 93]; 

STATE:
normal:{ n$7, error, this, e }, exn:{ }

INSTR=  _=*n$7:java.lang.AssertionError [line 93]; 

STATE UNCHANGED

INSTR=  n$7=*&amp;error:java.lang.AssertionError* [line 93]; 

STATE:
normal:{ error, this, e }, exn:{ }

</LISTING>
<hr width="100%">
<a name="session47" href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9.html#session47">node9#session47</a>(<a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>) [compute pre] nullify
<LISTING class='color_black'></LISTING>
<hr width="100%">
<a name="session48" href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9.html#session48">node9#session48</a>(<a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>) [exec] nullify
<LISTING class='color_black'>PRE STATE:
(normal:{ error, e }, exn:{ },normal:{ n$5, n$6 }, exn:{ })

INSTR=  n$7=*&amp;error:java.lang.AssertionError* [line 93]; 

STATE:
(normal:{ n$7, error, e }, exn:{ },normal:{ n$5, n$6 }, exn:{ })

INSTR=  _=*n$7:java.lang.AssertionError [line 93]; 

STATE:
(normal:{ _, n$7, error, e }, exn:{ },normal:{ n$5, n$6 }, exn:{ })

INSTR=  n$9=*&amp;e:java.lang.Object* [line 93]; 

STATE:
(normal:{ _, n$7, n$9, error, e }, exn:{ },normal:{ n$5, n$6 }, exn:{ })

INSTR=  n$10=_fun_Throwable AssertionError.initCause(Throwable)(n$7:java.lang.AssertionError*,n$9:java.lang.Object*) virtual [line 93]; 

STATE:
(normal:{ _, n$7, n$9, n$10, error, e }, exn:{ },
normal:{ n$5, n$6 }, exn:{ })

INSTR=  *&amp;$irvar1:java.lang.Throwable*=n$10 [line 93]; 

STATE:
(normal:{ error }, exn:{ },normal:{ _, n$7, n$9, n$10, $irvar1, e }, exn:{ })

</LISTING>
<hr width="100%">
<a name="session116" href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9.html#session116">node9#session116</a>(<a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>) [exec] devirtualizer
<LISTING class='color_black'></LISTING>
<hr width="100%">
<a name="session117" href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9.html#session117">node9#session117</a>(<a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>) [exec] devirtualizer
<LISTING class='color_black'></LISTING>
<hr width="100%">
<a name="session118" href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9.html#session118">node9#session118</a>(<a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>) [exec] devirtualizer
<LISTING class='color_black'></LISTING>
<hr width="100%">
<a name="session119" href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9.html#session119">node9#session119</a>(<a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>) [exec] devirtualizer
<LISTING class='color_black'>virtual call Throwable AssertionError.initCause(Throwable) 
</LISTING>
<hr width="100%">
<a name="session120" href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9.html#session120">node9#session120</a>(<a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>) [exec] devirtualizer
<LISTING class='color_black'></LISTING>
<hr width="100%">
<a name="session121" href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9.html#session121">node9#session121</a>(<a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>) [exec] devirtualizer
<LISTING class='color_black'></LISTING>
<hr width="100%">
<a name="session122" href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9.html#session122">node9#session122</a>(<a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>) [exec] devirtualizer
<LISTING class='color_black'></LISTING>
<hr width="100%">
<a name="session123" href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9.html#session123">node9#session123</a>(<a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>) [exec] devirtualizer
<LISTING class='color_black'></LISTING>
<hr width="100%">
<a name="session8" href="./../nodes/org.junit.rules.ErrorCollector.checkSucceeds%28org.junit.rules.ErrorCollectorTest%24CheckCallableThatThr.3cf67ad5a215a2e1984ee77b4479bc6e_node9.html#session8">node9#session8</a>(<a href="./../../ErrorCollector.java.572ca47a97c2ffbb3a38d177cca6c52b.html#LINE93">93</a>) biabduction
<LISTING class='color_black'>**** [FP:Pending] Object ErrorCollector.checkSucceeds(ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1) Node: 9, Procedure: Object ErrorCollector.checkSucceeds(ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1), Todo: 1 ****
  PROP 1 of 1:
  <span class='color_blue'><span class='color_red'>e != 0</span>; <span class='color_red'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_red'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e)</span>; <span class='color_red'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$2)</span>; <span class='color_red'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (_t$4)</span> ;
  <span class='color_red'>error = _t$3<span class='color_orange'>update:92</span>:java.lang.AssertionError*</span>; <span class='color_red'>CatchVar31 = _t$2<span class='color_orange'>initial</span>:void</span>; <span class='color_red'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_red'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_red'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_red'>old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_red'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_red'>$irvar1 = _t$1<span class='color_orange'>initial</span>:void</span>; <span class='color_red'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_red'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_red'>e = _t$0<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException*</span> ;
  <span class='color_red'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_red'>e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException</span>; <span class='color_red'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
  [footprint
     <span class='color_red'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$2<span class='color_orange'>none</span>:void( sub )</span>; <span class='color_red'>callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_red'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
     <span class='color_red'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
  .... Instructions: ....
  <span class='color_green'>n$7=*&amp;error:java.lang.AssertionError* [line 93];
  _=*n$7:java.lang.AssertionError [line 93];
  n$9=*&amp;e:java.lang.Object* [line 93];
  n$10=_fun_Throwable AssertionError.initCause(Throwable)(n$7:java.lang.AssertionError*,n$9:java.lang.Object*) virtual [line 93];
  *&amp;$irvar1:java.lang.Throwable*=n$10 [line 93];
  NULLIFY(&amp;$irvar1); [line 93];
  NULLIFY(&amp;e); [line 93];
  EXIT_SCOPE(_,n$7,n$9,n$10,$irvar1,e); [line 93];
  </span>
  
  Processing prop 1/1
    Instruction <span class='color_green'>n$7=*&amp;error:java.lang.AssertionError* [line 93]</span>
    in vpath_find exp:n$105
    vpath_find: found e :org.junit.internal.AssumptionViolatedException*
    in vpath_find exp:n$104
    vpath_find: found error :java.lang.AssertionError*
    .... Rearrangement Start ....
    Exp: &amp;error
    Prop:
    <span class='color_blue'>e != 0; MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e); MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error); UND &lt; AssertionError.<init>(Object) &gt; :92(@f$2); RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (n$108) ;
    error = n$104<span class='color_orange'>update:92</span>:java.lang.AssertionError*; CatchVar31 = n$106<span class='color_orange'>initial</span>:void; CatchVar7 = 0<span class='color_orange'>nullify</span>:void; $irvar0 = 0<span class='color_orange'>nullify</span>:void; callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); return = null<span class='color_orange'>update:91</span>:java.lang.Object*; $irvar1 = n$107<span class='color_orange'>initial</span>:void; this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); e = n$105<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException* ;
    e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException; error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError; old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )
    [footprint
       $RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$2<span class='color_orange'>none</span>:void( sub ); callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ) ;
       old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )  ]</span>
    
    in prop_iter_extend_ptsto lexp: &amp;error
    prop before:
    <span class='color_blue'>e != 0; MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e); MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error); UND &lt; AssertionError.<init>(Object) &gt; :92(@f$2); RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (n$108) ;
    error = n$104<span class='color_orange'>update:92</span>:java.lang.AssertionError*; CatchVar31 = n$106<span class='color_orange'>initial</span>:void; CatchVar7 = 0<span class='color_orange'>nullify</span>:void; $irvar0 = 0<span class='color_orange'>nullify</span>:void; callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); return = null<span class='color_orange'>update:91</span>:java.lang.Object*; $irvar1 = n$107<span class='color_orange'>initial</span>:void; this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); e = n$105<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException* ;
    e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException; error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError; old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )
    [footprint
       $RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$2<span class='color_orange'>none</span>:void( sub ); callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ) ;
       old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )  ]</span>
    
    
    prop list after:
    PROP 1 of 1:
    <span class='color_blue'><span class='color_blue'>e != 0</span>; <span class='color_blue'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e)</span>; <span class='color_blue'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_blue'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$2)</span>; <span class='color_blue'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (n$108)</span> ;
    <span class='color_blue'>error = n$104<span class='color_orange'>update:92</span>:java.lang.AssertionError*</span>; <span class='color_blue'>CatchVar31 = n$106<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_blue'>$irvar1 = n$107<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>e = n$105<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException*</span> ;
    <span class='color_blue'>e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException</span>; <span class='color_blue'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
    [footprint
       <span class='color_blue'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$2<span class='color_orange'>none</span>:void( sub )</span>; <span class='color_blue'>callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
       <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
    
    
    
    Instruction Returns
    PROP 1 of 1:
    <span class='color_blue'><span class='color_blue'>e != 0</span>; <span class='color_blue'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e)</span>; <span class='color_red'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_blue'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$2)</span>; <span class='color_blue'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (_t$3)</span> ;
    <span class='color_blue'>error = <span class='color_red'>n$7<span class='color_orange'>update:92</span></span>:java.lang.AssertionError*</span>; <span class='color_blue'>CatchVar31 = _t$2<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_blue'>$irvar1 = _t$1<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>e = _t$0<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException*</span> ;
    <span class='color_blue'>e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException</span>; <span class='color_red'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
    [footprint
       <span class='color_blue'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$2<span class='color_orange'>none</span>:void( sub )</span>; <span class='color_blue'>callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
       <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
    
    Instruction <span class='color_green'>_=*n$7:java.lang.AssertionError [line 93]</span>
    in vpath_find exp:n$109
    vpath_find: found e :org.junit.internal.AssumptionViolatedException*
    in vpath_find exp:n$7
    vpath_find: found error :java.lang.AssertionError*
    .... Rearrangement Start ....
    Exp: n$7
    Prop:
    <span class='color_blue'>e != 0; MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e); MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error); UND &lt; AssertionError.<init>(Object) &gt; :92(@f$2); RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (n$112) ;
    error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*; CatchVar31 = n$110<span class='color_orange'>initial</span>:void; CatchVar7 = 0<span class='color_orange'>nullify</span>:void; $irvar0 = 0<span class='color_orange'>nullify</span>:void; callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); return = null<span class='color_orange'>update:91</span>:java.lang.Object*; $irvar1 = n$111<span class='color_orange'>initial</span>:void; this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); e = n$109<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException* ;
    e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException; error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError; old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )
    [footprint
       $RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$2<span class='color_orange'>none</span>:void( sub ); callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ) ;
       old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )  ]</span>
    
    in prop_iter_extend_ptsto lexp: n$7
    prop before:
    <span class='color_blue'>e != 0; MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e); MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error); UND &lt; AssertionError.<init>(Object) &gt; :92(@f$2); RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (n$112) ;
    error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*; CatchVar31 = n$110<span class='color_orange'>initial</span>:void; CatchVar7 = 0<span class='color_orange'>nullify</span>:void; $irvar0 = 0<span class='color_orange'>nullify</span>:void; callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); return = null<span class='color_orange'>update:91</span>:java.lang.Object*; $irvar1 = n$111<span class='color_orange'>initial</span>:void; this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); e = n$109<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException* ;
    e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException; error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError; old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )
    [footprint
       $RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$2<span class='color_orange'>none</span>:void( sub ); callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ) ;
       old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )  ]</span>
    
    
    prop list after:
    PROP 1 of 1:
    <span class='color_blue'><span class='color_blue'>e != 0</span>; <span class='color_blue'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e)</span>; <span class='color_blue'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_blue'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$2)</span>; <span class='color_blue'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (n$112)</span> ;
    <span class='color_blue'>error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*</span>; <span class='color_blue'>CatchVar31 = n$110<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_blue'>$irvar1 = n$111<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>e = n$109<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException*</span> ;
    <span class='color_blue'>e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException</span>; <span class='color_blue'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
    [footprint
       <span class='color_blue'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$2<span class='color_orange'>none</span>:void( sub )</span>; <span class='color_blue'>callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
       <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
    
    
    
    Instruction Returns
    PROP 1 of 1:
    <span class='color_blue'><span class='color_blue'>e != 0</span>; <span class='color_blue'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e)</span>; <span class='color_blue'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_blue'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$2)</span>; <span class='color_blue'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (_t$3)</span> ;
    <span class='color_blue'>error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*</span>; <span class='color_blue'>CatchVar31 = _t$2<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_blue'>$irvar1 = _t$1<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>e = _t$0<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException*</span> ;
    <span class='color_blue'>e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException</span>; <span class='color_blue'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
    [footprint
       <span class='color_blue'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$2<span class='color_orange'>none</span>:void( sub )</span>; <span class='color_blue'>callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
       <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
    
    Instruction <span class='color_green'>n$9=*&amp;e:java.lang.Object* [line 93]</span>
    in vpath_find exp:n$113
    vpath_find: found e :org.junit.internal.AssumptionViolatedException*
    in vpath_find exp:n$7
    vpath_find: found error :java.lang.AssertionError*
    .... Rearrangement Start ....
    Exp: &amp;e
    Prop:
    <span class='color_blue'>e != 0; MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e); MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error); UND &lt; AssertionError.<init>(Object) &gt; :92(@f$2); RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (n$116) ;
    error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*; CatchVar31 = n$114<span class='color_orange'>initial</span>:void; CatchVar7 = 0<span class='color_orange'>nullify</span>:void; $irvar0 = 0<span class='color_orange'>nullify</span>:void; callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); return = null<span class='color_orange'>update:91</span>:java.lang.Object*; $irvar1 = n$115<span class='color_orange'>initial</span>:void; this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); e = n$113<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException* ;
    e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException; error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError; old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )
    [footprint
       $RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$2<span class='color_orange'>none</span>:void( sub ); callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ) ;
       old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )  ]</span>
    
    in prop_iter_extend_ptsto lexp: &amp;e
    prop before:
    <span class='color_blue'>e != 0; MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e); MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error); UND &lt; AssertionError.<init>(Object) &gt; :92(@f$2); RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (n$116) ;
    error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*; CatchVar31 = n$114<span class='color_orange'>initial</span>:void; CatchVar7 = 0<span class='color_orange'>nullify</span>:void; $irvar0 = 0<span class='color_orange'>nullify</span>:void; callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); return = null<span class='color_orange'>update:91</span>:java.lang.Object*; $irvar1 = n$115<span class='color_orange'>initial</span>:void; this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); e = n$113<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException* ;
    e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException; error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError; old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )
    [footprint
       $RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$2<span class='color_orange'>none</span>:void( sub ); callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ) ;
       old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )  ]</span>
    
    
    prop list after:
    PROP 1 of 1:
    <span class='color_blue'><span class='color_blue'>e != 0</span>; <span class='color_blue'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e)</span>; <span class='color_blue'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_blue'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$2)</span>; <span class='color_blue'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (n$116)</span> ;
    <span class='color_blue'>error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*</span>; <span class='color_blue'>CatchVar31 = n$114<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_blue'>$irvar1 = n$115<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>e = n$113<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException*</span> ;
    <span class='color_blue'>e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException</span>; <span class='color_blue'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
    [footprint
       <span class='color_blue'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$2<span class='color_orange'>none</span>:void( sub )</span>; <span class='color_blue'>callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
       <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
    
    
    
    Instruction Returns
    PROP 1 of 1:
    <span class='color_blue'><span class='color_red'>e != 0</span>; <span class='color_red'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e)</span>; <span class='color_blue'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_blue'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$2)</span>; <span class='color_red'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (_t$2)</span> ;
    <span class='color_blue'>error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*</span>; <span class='color_blue'>CatchVar31 = <span class='color_red'>_t$1<span class='color_orange'>initial</span></span>:void</span>; <span class='color_blue'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_blue'>$irvar1 = <span class='color_red'>_t$0<span class='color_orange'>initial</span></span>:void</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>e = <span class='color_red'>n$9<span class='color_orange'>update:91</span></span>:org.junit.internal.AssumptionViolatedException*</span> ;
    <span class='color_red'>e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException</span>; <span class='color_blue'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
    [footprint
       <span class='color_blue'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$2<span class='color_orange'>none</span>:void( sub )</span>; <span class='color_blue'>callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
       <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
    
    Instruction <span class='color_green'>n$10=_fun_Throwable AssertionError.initCause(Throwable)(n$7:java.lang.AssertionError*,n$9:java.lang.Object*) virtual [line 93]</span>
    in vpath_find exp:n$9
    vpath_find: found e :org.junit.internal.AssumptionViolatedException*
    in vpath_find exp:n$7
    vpath_find: found error :java.lang.AssertionError*
    Couldn't find method in the hierarchy of type java.lang.AssertionError
    Skipping function 'Throwable AssertionError.initCause(AssumptionViolatedException)': unknown method
    <span class='color_orange'>WARNING: dropping non-footprint </span>n$10 = @f$3
    Instruction Returns
    PROP 1 of 1:
    <span class='color_blue'><span class='color_red'>n$10 = @f$2</span> ;
    <span class='color_blue'>e != 0</span>; <span class='color_blue'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e)</span>; <span class='color_blue'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_red'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$3)</span>; <span class='color_blue'>UND &lt; Throwable AssertionError.initCause(AssumptionViolatedException) &gt; :93(@f$2)</span>; <span class='color_blue'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (_t$2)</span> ;
    <span class='color_blue'>error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*</span>; <span class='color_blue'>CatchVar31 = _t$1<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_blue'>$irvar1 = _t$0<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>e = n$9<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException*</span> ;
    <span class='color_blue'>e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException</span>; <span class='color_blue'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
    [footprint
       <span class='color_blue'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = <span class='color_red'>@f$3<span class='color_orange'>none</span></span>:void( sub )</span>; <span class='color_red'>$RET_java.lang.AssertionError.initCause(org.junit.internal.AssumptionViolatedException):java.lang.Throwable|abducedRetvar = @f$2<span class='color_orange'>none</span>:java.lang.Throwable*( sub )</span>; <span class='color_blue'>callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
       <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
    
    Instruction <span class='color_green'>*&amp;$irvar1:java.lang.Throwable*=n$10 [line 93]</span>
    in vpath_find exp:n$9
    vpath_find: found e :org.junit.internal.AssumptionViolatedException*
    in vpath_find exp:n$7
    vpath_find: found error :java.lang.AssertionError*
    .... Rearrangement Start ....
    Exp: &amp;$irvar1
    Prop:
    <span class='color_blue'>n$10 = @f$2 ;
    e != 0; MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e); MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error); UND &lt; AssertionError.<init>(Object) &gt; :92(@f$3); UND &lt; Throwable AssertionError.initCause(AssumptionViolatedException) &gt; :93(@f$2); RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (n$122) ;
    error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*; CatchVar31 = n$120<span class='color_orange'>initial</span>:void; CatchVar7 = 0<span class='color_orange'>nullify</span>:void; $irvar0 = 0<span class='color_orange'>nullify</span>:void; callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); return = null<span class='color_orange'>update:91</span>:java.lang.Object*; $irvar1 = n$121<span class='color_orange'>initial</span>:void; this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); e = n$9<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException* ;
    e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException; error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError; old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )
    [footprint
       $RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$3<span class='color_orange'>none</span>:void( sub ); $RET_java.lang.AssertionError.initCause(org.junit.internal.AssumptionViolatedException):java.lang.Throwable|abducedRetvar = @f$2<span class='color_orange'>none</span>:java.lang.Throwable*( sub ); callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ) ;
       old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )  ]</span>
    
    in prop_iter_extend_ptsto lexp: &amp;$irvar1
    prop before:
    <span class='color_blue'>n$10 = @f$2 ;
    e != 0; MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e); MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error); UND &lt; AssertionError.<init>(Object) &gt; :92(@f$3); UND &lt; Throwable AssertionError.initCause(AssumptionViolatedException) &gt; :93(@f$2); RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (n$122) ;
    error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*; CatchVar31 = n$120<span class='color_orange'>initial</span>:void; CatchVar7 = 0<span class='color_orange'>nullify</span>:void; $irvar0 = 0<span class='color_orange'>nullify</span>:void; callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); return = null<span class='color_orange'>update:91</span>:java.lang.Object*; $irvar1 = n$121<span class='color_orange'>initial</span>:void; this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); e = n$9<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException* ;
    e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException; error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError; old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )
    [footprint
       $RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$3<span class='color_orange'>none</span>:void( sub ); $RET_java.lang.AssertionError.initCause(org.junit.internal.AssumptionViolatedException):java.lang.Throwable|abducedRetvar = @f$2<span class='color_orange'>none</span>:java.lang.Throwable*( sub ); callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ) ;
       old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )  ]</span>
    
    
    prop list after:
    PROP 1 of 1:
    <span class='color_blue'><span class='color_blue'>n$10 = @f$2</span> ;
    <span class='color_blue'>e != 0</span>; <span class='color_blue'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e)</span>; <span class='color_blue'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_blue'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$3)</span>; <span class='color_blue'>UND &lt; Throwable AssertionError.initCause(AssumptionViolatedException) &gt; :93(@f$2)</span>; <span class='color_blue'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (n$122)</span> ;
    <span class='color_blue'>error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*</span>; <span class='color_blue'>CatchVar31 = n$120<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_blue'>$irvar1 = n$121<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>e = n$9<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException*</span> ;
    <span class='color_blue'>e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException</span>; <span class='color_blue'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
    [footprint
       <span class='color_blue'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$3<span class='color_orange'>none</span>:void( sub )</span>; <span class='color_blue'>$RET_java.lang.AssertionError.initCause(org.junit.internal.AssumptionViolatedException):java.lang.Throwable|abducedRetvar = @f$2<span class='color_orange'>none</span>:java.lang.Throwable*( sub )</span>; <span class='color_blue'>callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
       <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
    
    
    
    Instruction Returns
    PROP 1 of 1:
    <span class='color_blue'><span class='color_blue'>n$10 = <span class='color_red'>$irvar1</span></span> ;
    <span class='color_blue'>e != 0</span>; <span class='color_blue'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e)</span>; <span class='color_blue'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_blue'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$3)</span>; <span class='color_red'>UND &lt; Throwable AssertionError.initCause(AssumptionViolatedException) &gt; :93($irvar1)</span>; <span class='color_red'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (_t$1)</span> ;
    <span class='color_blue'>error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*</span>; <span class='color_blue'>CatchVar31 = <span class='color_red'>_t$0<span class='color_orange'>initial</span></span>:void</span>; <span class='color_blue'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>old_callable = <span class='color_red'>@f$2<span class='color_orange'>formal</span></span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_blue'>$irvar1 = <span class='color_red'>@f$1<span class='color_orange'>update:93</span></span>:void</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>e = n$9<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException*</span> ;
    <span class='color_blue'>e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException</span>; <span class='color_blue'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_red'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
    [footprint
       <span class='color_blue'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$3<span class='color_orange'>none</span>:void( sub )</span>; <span class='color_blue'>$RET_java.lang.AssertionError.initCause(org.junit.internal.AssumptionViolatedException):java.lang.Throwable|abducedRetvar = <span class='color_red'>@f$1<span class='color_orange'>none</span></span>:java.lang.Throwable*( sub )</span>; <span class='color_blue'>callable = <span class='color_red'>@f$2<span class='color_orange'>formal</span></span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
       <span class='color_red'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
    
    Instruction <span class='color_green'>NULLIFY(&amp;$irvar1); [line 93]</span>
    in vpath_find exp:n$9
    vpath_find: found e :org.junit.internal.AssumptionViolatedException*
    in vpath_find exp:n$7
    vpath_find: found error :java.lang.AssertionError*
    Instruction Returns
    PROP 1 of 1:
    <span class='color_blue'><span class='color_blue'>n$10 = <span class='color_red'>@f$2</span></span> ;
    <span class='color_blue'>e != 0</span>; <span class='color_blue'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(e)</span>; <span class='color_blue'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_blue'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$3)</span>; <span class='color_red'>UND &lt; Throwable AssertionError.initCause(AssumptionViolatedException) &gt; :93(@f$2)</span>; <span class='color_blue'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (_t$1)</span> ;
    <span class='color_blue'>error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*</span>; <span class='color_blue'>CatchVar31 = _t$0<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>old_callable = <span class='color_red'>@f$1<span class='color_orange'>formal</span></span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_blue'>$irvar1 = <span class='color_red'>0<span class='color_orange'>nullify</span></span>:void</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>e = n$9<span class='color_orange'>update:91</span>:org.junit.internal.AssumptionViolatedException*</span> ;
    <span class='color_blue'>e|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException</span>; <span class='color_blue'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_red'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
    [footprint
       <span class='color_blue'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$3<span class='color_orange'>none</span>:void( sub )</span>; <span class='color_blue'>$RET_java.lang.AssertionError.initCause(org.junit.internal.AssumptionViolatedException):java.lang.Throwable|abducedRetvar = <span class='color_red'>@f$2<span class='color_orange'>none</span></span>:java.lang.Throwable*( sub )</span>; <span class='color_blue'>callable = <span class='color_red'>@f$1<span class='color_orange'>formal</span></span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
       <span class='color_red'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
    
    Instruction <span class='color_green'>NULLIFY(&amp;e); [line 93]</span>
    in vpath_find exp:n$9
    vpath_find: found e :org.junit.internal.AssumptionViolatedException*
    in vpath_find exp:n$7
    vpath_find: found error :java.lang.AssertionError*
    Instruction Returns
    PROP 1 of 1:
    <span class='color_blue'><span class='color_blue'>n$10 = @f$2</span> ;
    <span class='color_blue'>n$9 != 0</span>; <span class='color_blue'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(n$9)</span>; <span class='color_blue'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_blue'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$3)</span>; <span class='color_blue'>UND &lt; Throwable AssertionError.initCause(AssumptionViolatedException) &gt; :93(@f$2)</span>; <span class='color_blue'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (_t$1)</span> ;
    <span class='color_blue'>error = n$7<span class='color_orange'>update:92</span>:java.lang.AssertionError*</span>; <span class='color_blue'>CatchVar31 = _t$0<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_blue'>$irvar1 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>e = <span class='color_red'>0<span class='color_orange'>nullify</span></span>:org.junit.internal.AssumptionViolatedException*</span> ;
    <span class='color_blue'>n$9|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException</span>; <span class='color_blue'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
    [footprint
       <span class='color_blue'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$3<span class='color_orange'>none</span>:void( sub )</span>; <span class='color_blue'>$RET_java.lang.AssertionError.initCause(org.junit.internal.AssumptionViolatedException):java.lang.Throwable|abducedRetvar = @f$2<span class='color_orange'>none</span>:java.lang.Throwable*( sub )</span>; <span class='color_blue'>callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
       <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
    
    Instruction <span class='color_green'>EXIT_SCOPE(_,n$7,n$9,n$10,$irvar1,e); [line 93]</span>
    in vpath_find exp:n$9
    vpath_find: cannot find n$9
    in vpath_find exp:n$7
    vpath_find: found error :java.lang.AssertionError*
    .... Prop with garbage in normal part ....
      PROP:
      <span class='color_blue'>_t$143 != 0; MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(_t$143); MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error); UND &lt; AssertionError.<init>(Object) &gt; :92(@f$3); UND &lt; Throwable AssertionError.initCause(AssumptionViolatedException) &gt; :93(@f$2); RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (_t$1) ;
      error = _t$142<span class='color_orange'>update:92</span>:java.lang.AssertionError*; CatchVar31 = _t$0<span class='color_orange'>initial</span>:void; CatchVar7 = 0<span class='color_orange'>nullify</span>:void; $irvar0 = 0<span class='color_orange'>nullify</span>:void; callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); return = null<span class='color_orange'>update:91</span>:java.lang.Object*; $irvar1 = 0<span class='color_orange'>nullify</span>:void; this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ); e = 0<span class='color_orange'>nullify</span>:org.junit.internal.AssumptionViolatedException* ;
      _t$143|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException; error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError; old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )
      [footprint
         $RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$3<span class='color_orange'>none</span>:void( sub ); $RET_java.lang.AssertionError.initCause(org.junit.internal.AssumptionViolatedException):java.lang.Throwable|abducedRetvar = @f$2<span class='color_orange'>none</span>:java.lang.Throwable*( sub ); callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub ); this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub ) ;
         old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )  ]</span>
      PREDICATE:
      _t$143|->{fAssumption:0<span class='color_orange'>alloc</span>, fMatcher:0<span class='color_orange'>alloc</span>, fValue:0<span class='color_orange'>alloc</span>, fValueMatcher:0<span class='color_orange'>alloc</span>}<span class='color_orange'>alloc</span>:org.junit.internal.AssumptionViolatedException
      ATTRIBUTE: MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>
      explain_leak: case not matched in instr <span class='color_green'>EXIT_SCOPE(_,n$7,n$9,n$10,$irvar1,e); [line 93]</span>
    Instruction Returns
    PROP 1 of 1:
    <span class='color_blue'><span class='color_red'>_t$2 != 0</span>; <span class='color_red'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(_t$2)</span>; <span class='color_blue'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_blue'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$3)</span>; <span class='color_blue'>UND &lt; Throwable AssertionError.initCause(AssumptionViolatedException) &gt; :93(@f$2)</span>; <span class='color_red'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (_t$3)</span> ;
    <span class='color_blue'>error = <span class='color_red'>_t$1<span class='color_orange'>update:92</span></span>:java.lang.AssertionError*</span>; <span class='color_blue'>CatchVar31 = _t$0<span class='color_orange'>initial</span>:void</span>; <span class='color_blue'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_blue'>$irvar1 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>e = 0<span class='color_orange'>nullify</span>:org.junit.internal.AssumptionViolatedException*</span> ;
    <span class='color_red'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
    [footprint
       <span class='color_blue'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = @f$3<span class='color_orange'>none</span>:void( sub )</span>; <span class='color_blue'>$RET_java.lang.AssertionError.initCause(org.junit.internal.AssumptionViolatedException):java.lang.Throwable|abducedRetvar = @f$2<span class='color_orange'>none</span>:java.lang.Throwable*( sub )</span>; <span class='color_blue'>callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
       <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
    
    .... After Symbolic Execution ....
    PROP 1 of 1:
    <span class='color_blue'><span class='color_red'>_t$2 != 0</span>; <span class='color_red'>MEMne &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call():90 &gt; <span class='color_orange'>{vpath: }</span>(_t$2)</span>; <span class='color_red'>MEMne &lt; new:92 &gt; <span class='color_orange'>{vpath: }</span>(error)</span>; <span class='color_red'>UND &lt; AssertionError.<init>(Object) &gt; :92(@f$3)</span>; <span class='color_blue'>UND &lt; Throwable AssertionError.initCause(AssumptionViolatedException) &gt; :93(@f$2)</span>; <span class='color_blue'>RET &lt; Object ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1.call() &gt; (_t$3)</span> ;
    <span class='color_blue'>error = <span class='color_red'>_t$1<span class='color_orange'>update:92</span></span>:java.lang.AssertionError*</span>; <span class='color_blue'>CatchVar31 = <span class='color_red'>_t$0<span class='color_orange'>initial</span></span>:void</span>; <span class='color_blue'>CatchVar7 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>$irvar0 = 0<span class='color_orange'>nullify</span>:void</span>; <span class='color_blue'>callable = 0<span class='color_orange'>nullify</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>old_callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>return = null<span class='color_orange'>update:91</span>:java.lang.Object*</span>; <span class='color_blue'>$irvar1 = <span class='color_red'>0<span class='color_orange'>nullify</span></span>:void</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>old_this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span>; <span class='color_blue'>e = <span class='color_red'>0<span class='color_orange'>nullify</span></span>:org.junit.internal.AssumptionViolatedException*</span> ;
    <span class='color_red'>error|->{}<span class='color_orange'>alloc</span>:java.lang.AssertionError</span>; <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>
    [footprint
       <span class='color_blue'>$RET_java.lang.AssertionError.<init>(java.lang.Object)|abducedRetvar = <span class='color_red'>@f$3<span class='color_orange'>none</span></span>:void( sub )</span>; <span class='color_red'>$RET_java.lang.AssertionError.initCause(org.junit.internal.AssumptionViolatedException):java.lang.Throwable|abducedRetvar = @f$2<span class='color_orange'>none</span>:java.lang.Throwable*( sub )</span>; <span class='color_blue'>callable = @f$1<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1*( sub )</span>; <span class='color_blue'>this = @f$0<span class='color_orange'>formal</span>:org.junit.rules.ErrorCollector*( sub )</span> ;
       <span class='color_blue'>old_callable|->{}<span class='color_orange'>rearrange:(z)90</span>:org.junit.rules.ErrorCollectorTest$CheckCallableThatThrowsInternalAssumptionViolatedException$1( sub )</span>  ]</span>
    
    
    
  </LISTING>